### DISCUSSION TITLE
when i place Space between word in svg it show in UI but when i convert from svg to pdf by using Magick.NET it remove space

### DISCUSSION THREAD

**Discussion by ShrikantShinde1991** [dlemstra/Magick.NET#1330] [Help]
![No18_Test](https://user-images.githubusercontent.com/121941277/211462992-4ddf4396-aed0-4ec1-93eb-0c68b1de738a.svg)

**Comment by ShrikantShinde1991** [comment]
Above file is my svg file. I already place space between word but after converting svg to pdf by using Magick.NET dll it removes all space between word or before word or after word.

**Comment by dlemstra** [comment]
It looks like the library that is being used (`librsvg`) does not support `xml:space="preserve"`. I would advise you to install Inkscape on your machine instead to get a better result. 

I am also not sure if this library is the best library to do this conversion. You might want to do some research and find a better product for this situation.
